app/src/main/java/cm/aptoide/pt/ApplicationModule.java

```diff
@@ -1018,6 +1009,28 @@ AnalyticsBodyInterceptorV7 provideAnalyticsBodyInterceptorV7(
     return new Database();
   }
 
+  @Singleton @Provides AptoideDatabase providesAptoideDataBase() {
+    return Room.databaseBuilder(getApplicationContext(), AptoideDatabase.class,
+        BuildConfig.ROOM_DATABASE_NAME)
+        .fallbackToDestructiveMigration()
```

Comment:
This should only be used for testing, we cannot forget to remove before it is merged to dev.